godot-ffi/src/opaque.rs

```diff
@@ -11,7 +11,8 @@
 ///
 /// Note: due to `align(8)` and not `packed` repr, this type may be bigger than `N` bytes
```

Comment:
Comment would need to be updated:
```suggestion
/// Note: due to `align(4)`/ `align(8)` and not `packed` repr, this type may be bigger than `N` bytes
```